src/main/java/teammates/ui/webapi/action/GetSessionResultsAction.java

```diff
@@ -25,10 +25,6 @@ protected AuthType getMinAuthLevel() {
 
     @Override
     public void checkSpecificAccessControl() {
-        if (userInfo.isAdmin) {
-            return;
-        }
-
```

Comment:
I'm assuming that with the new auth flow you can't access the results page as an admin? Just confirming, because this guard clause is being fixed in #9994

Reply:
There is no reason for admin to have this elevated privilege.